fastlane/Fastfile

```diff
@@ -584,21 +584,15 @@ end
 
 desc "Ensure the correct formatting for the fastlane tools"
 private_lane :ensure_tool_name_formatting do
+  test_tool_name_formatting_ensurance_lane
```

Comment:
Now that we have a spec, this should be removed.

Reply:
Oops, forgot to commit that part :) it's also missing the require